frontend/src/app/home/Pet/Actions.tsx

```diff
@@ -160,4 +283,7 @@ export function Actions({
 const actionDescriptions: Record<PetAction, string> = {
   feed: "Feeding your pet will boost its Energy Points...",
   play: "Playing with your pet will make it happy and consume its Energy Points...",
+  buy: "Buy an accessory for your pet...",
+  wear: "Wear an accessory for your pet...",
+  unwear: "Take off an accessory for your pet...",
```

Comment:
These can use some touch from ChatGPT. For example starting with Buying, Wearing, and Taking off, Following the feed and play options.